packages/zoltan2/src/input/Zoltan2_MeshAdapter.hpp

```diff
@@ -127,6 +127,7 @@ class MeshAdapter : public BaseAdapter<User> {
 
 #ifndef DOXYGEN_SHOULD_SKIP_THIS
   typedef typename InputTraits<User>::scalar_t scalar_t;
+  typedef typename InputTraits<User>::lno_t offset_t; // default to lno_t
```

Comment:
I know I said to make this default assignment to offset_t, but I wonder now whether this decision will come back to burn us later (e.g., a user defines offset_t != lno_t in InputTraits and then wonders why his offset_t isn't being used).  
Would there be any harm in using InputTraits<User>::offset_t for offset_t here?  I will check with our MeshAdapter users to make sure everything is OK with them.